README.md

```diff
@@ -23,3 +23,19 @@ inefficient.
 
 What is the big $\Theta$ complexity of your implementation? Add your
 answer, including your reasoning, to this markdown file.
+
+tests i made up and hand solved:
+![tests.img](image.png)
+
+my implementation has 2 main nested forloops with a priority que at the top of all of those.
+The priority queue at worst will have to loop though all of the items inside to add to the queue but this will only ever happend a maximum of |v| times because every node in the graph is pushed to the queue only once so its a |v| ammount of work. on average though a priority queue has a runtime of log(|v|) because of the fact that we dequeue items from the queue all the time meaning that lots of the time there is less than |v| ammount of work to do.
```

Comment:
How do you get a log runtime for the priority queue? I don't see any log time operations there.